components/automate-ui/src/app/pages/team-details/team-details.component.spec.ts

```diff
@@ -37,13 +36,54 @@ import {
 import { Team } from 'app/entities/teams/team.model';
 import { TeamDetailsComponent } from './team-details.component';
 
-describe('TeamDetailsComponent', () => {
+const declarations: any[] = [
```

Comment:
Restructured these tests to have two describe blocks for version 'v2' and version 'v1'.